pkgs/top-level/python-packages.nix

```diff
@@ -31763,6 +31763,128 @@ EOF
+      platforms   = platforms.linux;
+    };
+    propagatedBuildInputs = with self; [ urllib3 pytest ] ++ (if isPy3k then [requests2] else [requests]);
+  };
+
+  pysigset = buildPythonPackage rec {
+    pname = "pysigset";
+    name = "${pname}-${version}";
+    version = "0.2.1";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/p/${pname}/${name}.tar.gz";
+      sha256 = "0f3g705r2zb09xvz81kmci4iza7ph2gqydmmv0nxqfs50msn3pw1";
+    };
+    meta = {
+      description = "Python blocking/suspending signals under Linux/OSX using ctypes sigprocmask access";
+      homepage    = "https://github.com/ossobv/pysigset";
+      license     = licenses.gpl;
+      platforms   = platforms.linux;
+    };
+  };
+
+  blessed = buildPythonPackage rec {
+    pname = "blessed";
+    name = "${pname}-${version}";
+    version = "1.14.1";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/b/${pname}/${name}.tar.gz";
+      sha256 = "0wy0na399swybp5bzys31pn57vg34bjsl0kv5zf496996gc8k8jq";
+    };
+    meta = {
+      description = "A thin, practical wrapper around terminal styling, screen positioning, and keyboard input.";
+      homepage    = "https://github.com/jquast/blessed";
+      license     = licenses.mit;
+      platforms   = platforms.linux;
+    };
+    buildInputs = with self; [ pytest mock ];
+    propagatedBuildInputs = with self; [ wcwidth six ];
+  };
+
+  Flask-RESTful = buildPythonPackage rec {
+    pname = "Flask-RESTful";
+    name = "${pname}-${version}";
+    version = "0.3.5";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/F/${pname}/${name}.tar.gz";
+      sha256 = "0hjcmdb56b7z4bkw848lxfkyrpnkwzmqn2dgnlv12mwvjpzsxr6c";
+    };
+    PYTHON_EGG_CACHE = "`pwd`/.egg-cache";
+    meta = {
+      description = "Simple framework for creating REST APIs";
+      homepage    = "https://github.com/flask-restful/flask-restful/";
+      license     = licenses.bsd3;
+      platforms   = platforms.linux;
+    };
+    propagatedBuildInputs = with self; [ flask pytz six aniso8601 itsdangerous click jinja2 werkzeug dateutil nose ];
+  };
+
+  aniso8601 = buildPythonPackage rec {
+    pname = "aniso8601";
+    name = "${pname}-${version}";
+    version = "1.2.0";
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/a/${pname}/${name}.tar.gz";
+      sha256 = "1m2d83rm684xdf54ynfd9lv3slv7bkqq6pcirh2aibvl4pw0092h";
+    };
+    meta = {
+      description = "A library for parsing ISO 8601 strings";
+      homepage    = "https://bitbucket.org/nielsenb/aniso8601";
+      license     = licenses.bsd3;
+      platforms   = platforms.linux;
+    };
+    propagatedBuildInputs = with self; [ dateutil ];
+  };
+
+  scruffington = buildPythonPackage rec {
+    pname = "scruffington";
+    name = "${pname}-${version}";
+    version = "0.3.7";
+    format = "wheel";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/6c/73/18d180df19d55e8803731ad638513038906914063bd9bde8eea3cfa1ece5/${name}-py2.py3-none-any.whl";
```

Comment:
please use url's like 
```
url = "mirror://pypi/a/${pname}/${name}.tar.gz";
```
or, even better, 
```
url = "mirror://pypi/${builtins.substring 0 1 pname}/${pname}/${name}.tar.gz";
```
The one you use now consists of a big hash which changes with every version requiring updating the url.